spec/support/rails_template.rb

```diff
@@ -58,7 +58,7 @@
 RUBY
 
 # Setup Active Admin
-generate 'active_admin:install'
+generate "active_admin:install#{" --use-webpacker" if ENV["BUNDLE_GEMFILE"] == File.expand_path("../../gemfiles/rails_60_webpacker/Gemfile", __dir__)}"
```

Comment:
Should we switch to webpacker as default for rails 6+, just like the rails team did?

/cc @deivid-rodriguez 

This line would then be more like
```rb
generate "active_admin:install#{" --use-webpacker" if rails >= 6}" # Rails::VERSION::MAJOR or smth, don't know by heart
```

Reply:
We can consider that in the future, but initially I'd like this to be optin.